### ISSUE TITLE
<ToastNotification /> prop timeout > 0 does not render component on UI.

### ISSUE THREAD

**Issue by SwapnilAhire** [carbon-design-system/carbon#8909]
BUG  - 
1. I am on @carbon/ibm-security version 1.41.0 and 
2. I am facing issue with <ToastNotification /> component. 

Description -
 - When I pass prop timeout value > 0 (say 12000), it is not rendering the component on UI.
 - BUT when I pass prop timeout as 0, then it is rendered as expected on UI.

```
<ToastNotification
    onCloseButtonClick={onToastClose}
    className='toast-notification animated slideInUp'
    title={error.title}
    subtitle={error.subtitle}
    kind={error.title.toLowerCase()}
    caption=''
    timeout={12000}
/>
```

Expected behavior -
 - Notification should be rendered on UI for provided timeout span

Actual behavior - 
 - Notification is not rendered at all if timeout prop value is > 0

Steps for reproducing -
(TRY it in both Chrome and Firefox)

Browsers
 - Not browser specific

Optional information
- None

**Comment by emyarod** [comment]
can you create a [reduced test case in Code Sandbox](https://codesandbox.io/s/github/carbon-design-system/carbon/tree/main/packages/react/examples/codesandbox) for easier testing and debugging? The timeout prop seems to function as expected in the Carbon Storybook

**Comment by palak1459** [comment]
@emyarod @SwapnilAhire https://codesandbox.io/s/hopeful-aryabhata-h824j?file=/src/index.js is the reduced test case. It's not working for 
"carbon/ibm-security": "1.40.0" and later. Also found out that this issue is already resolved in https://github.com/carbon-design-system/carbon/issues/8348 and may be available in next version.

**Comment by emyarod** [comment]
closing as resolved

**Comment by SwapnilAhire** [comment]
@emyarod Could you please help me with the list of packages that we need to update in order get this fix?

**Comment by emyarod** [comment]
the fix was released in `carbon-components-react@7.33.0`
